### DISCUSSION TITLE
IS IT POSSIBLE TO PLAY GTA 5 ON WISKEY EVEN AFTER ENABLING DXVK

### DISCUSSION THREAD

**Discussion by gamehunter1** [Whisky-App/Whisky#878] [Q&A] [answered]
guyssss i m soo tired even after watching thousands of videos i couldn't find any solution....i just purchased gta5 on steam and it wont start i tried enabling DXVK but the launcher just shows logo and then disappears and the game wont start also

<img width="906" alt="Screenshot 2024-03-20 at 4 18 49 PM" src="https://github.com/Whisky-App/Whisky/assets/164167550/e716c491-2a13-4367-abfd-68ada6de794b">

**Comment by hahayupgit** [comment, marked as answer]
https://docs.getwhisky.app/game-support/gta-5.html
<img width="1013" alt="Screenshot 2024-04-08 at 11 32 07 PM" src="https://github.com/Whisky-App/Whisky/assets/161992562/2db6396b-cc35-44d8-8c1a-e4330ff50a02">
